src/core/public/notifications/toasts/error_toast.tsx

```diff
@@ -42,6 +46,8 @@ const isRequestError = (e: Error | RequestError): e is RequestError => {
   return false;
 };
 
+const isQuotaExceededError = (e: Error): boolean => e.name === 'QuotaExceededError';
```

Comment:
this code is not generic enough to be here. Not sure that we should modify that file at all.